e2e-test/helper.py

```diff
@@ -27,49 +28,19 @@ def setup(bee_name):
     logging.debug(f"leo_url: {leo_url}")
     return workspace_manager_url, rawls_url, leo_url
 
-# CREATE WORKSPACE ACTION
-def create_workspace(cbas, billing_project_name, header, workspace_name = ""):
-    # create a new workspace, need to have attributes or api call doesnt work
-    rawls_workspace_api = f"{rawls_url}/api/workspaces";
-    workspace_name = workspace_name if workspace_name else f"api-workspace-{''.join(random.choices(string.ascii_lowercase, k=5))}"
-    request_body= {
-      "namespace": billing_project_name, # Billing project name
-      "name": workspace_name,
-      "attributes": {}};
-
-    logging.debug(f"Creating workspace with name {workspace_name}")
-    workpace_response = requests.post(url=rawls_workspace_api, json=request_body, headers=header)
-    logging.debug(f"url: {rawls_workspace_api}")
-    logging.debug(f"response: {workpace_response}")
-    logging.debug(f"request_body: {request_body}")
-    assert workpace_response.status_code == 201, f"Error creating workspace: ${workpace_response.text}"
-    
-    # example json that is returned by request:
-    # {
-    #   "attributes": {},
-    #   "authorizationDomain": [],
-    #   "bucketName": "",
-    #   "createdBy": "yulialovesterra@gmail.com",
-    #   "createdDate": "2023-08-03T20:10:59.116Z",
-    #   "googleProject": "",
-    #   "isLocked": False,
-    #   "lastModified": "2023-08-03T20:10:59.116Z",
-    #   "name": "api-workspace-1",
-    #   "namespace": "yuliadub-test2",
-    #   "workspaceId": "ac466322-2325-4f57-895d-fdd6c3f8c7ad",
-    #   "workspaceType": "mc",
-    #   "workspaceVersion": "v2"
-    # }
-    workspace_response_json = workpace_response.json()
-    logging.debug(f"json response: {workspace_response_json}")
-    data = json.loads(json.dumps(workspace_response_json))
-    
-    logging.debug(f"data['workspaceId']:  {data['workspaceId']}")
+# CREATE WORKSPACE ACTION WITH APP CREATION
+def create_workspace_with_cromwell_app(cbas, billing_project_name, azure_token, workspace_name = ""):
+    workspace_id, workspace_name = create_workspace(billing_project_name, azure_token, rawls_url, workspace_name)

```

Comment:
I think if cbas will have a separate test script then this can probably be refactored a bit to be more WDS-specific and not use cromwell or cbas at all.  Most or all of what's in here referring to cbas was sort of a placeholder for future tests.  But that can be a separate ticket that AJ can be in charge of.

Reply:
This method is creating the old Cromwell app type. I wasn't sure how the WDS script is using it or planning to use hence I didn't remove it. I agree that it can be a separate AJ ticket to refactor it into more WDS specific function or if WDS test doesn't need CBAS then `create_workspace` in `workspace_helper.py` can also be used in place of this.